core/db/db.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+const conectDb = async () => {
+  await mongoose
+    .connect("mongodb://localhost/todoapp", {
```

Comment:
You must get the mongoose `URI` from `process.env` environment variables.
so if you will to deploy this application in a host you should provide the MongoDB URI in environment variables.
example:
`await mongoose.connect(process.env.MONGO_URI);`